docs/basics/sso/configuring-sso-in-okta.md

```diff
@@ -1,42 +0,0 @@
----
-id: configuring-sso-in-okta
-title: Configuring SSO in Okta 
-sidebar_label: Configuring Okta
----
-
```

Comment:
Hi @przemek-sl 👋 ~ As an alternative to removing and re-adding the doc, you could keep the doc itself but remove it from the side nav menu and add the `noindex` tag so it doesn't show in search results:

```
<head>
  <meta name="robots" content="noindex" />
</head>
```

The doc will still be accessible through the direct link, so you can share the doc with the Okta team, if needed. If the page is removed, we may get an 404 error.

Reply:
@aliciakrebs ^FYI

Reply:
Yes, that would be a better approach.

Reply:
I thought about this solution but we have to delete this page completely, so that customers do not access it directly via URL. This way it will be less confusing for them. We are launching tomorrow the SSO migration wizard and Okta is still reviewing our new app. We do not want our customers to use the old app which is already in the Okta catalog. It's better to temporarily delete this tutorial, until the new app is published by Okta, which I hope should happen very soon.

Reply:
@lysannep is this good to merge based on @przemek-sl's notes?

Reply:
Yep - this is good to merge. Though in the meantime, we should add a redirect until the new app is published by Okta. 

@przemek-sl Can I add a temporary redirect to https://docs.saucelabs.com/basics/sso/setting-up-sso/ ?

Reply:
Yes, please add the redirect.